android/java/org/chromium/chrome/browser/crypto_wallet/fragments/EditVisibleAssetsBottomSheetDialogFragment.java

```diff
@@ -310,24 +314,53 @@ public void onClick(View v) {
 
     private class TextWatcherImpl implements TextWatcher {
         Dialog mDialog;
+        boolean selfChange;
```

Comment:
let's define it to false explicitly

Reply:
That will emit a warning I think, I will suppress it.

Reply:
Updated.